src/test.rs

```diff
@@ -114,7 +113,7 @@ lazy_static! {
 
 pub(crate) fn update_options_for_testing(options: &mut ClientOptions) {
     if options.max_pool_size.is_none() {
-        options.max_pool_size = Some(MAX_POOL_SIZE);
+        options.max_pool_size = Some(DEFAULT_MAX_POOL_SIZE);
```

Comment:
I think we can remove this logic entirely since the driver will already apply this default internally.

Reply:
Indeed! Done.